pkg/cvo/sync_worker.go

```diff
@@ -300,18 +301,18 @@ func (w *SyncWorker) syncPayload(ctx context.Context, work *SyncWork,
 			})
 			return nil, err
 		}
+		acceptedRisksMsg := ""
 		if info.VerificationError != nil {
-			msg := ""
 			for err := info.VerificationError; err != nil; err = errors.Unwrap(err) {
 				details := err.Error()
-				if !strings.Contains(msg, details) {
+				if !strings.Contains(acceptedRisksMsg, details) {
 					// library-go/pkg/verify wraps the details, but does not include them
 					// in the top-level error string.  If we have an error like that,
 					// include the details here.
-					msg = fmt.Sprintf("%s\n%s", msg, details)
+					acceptedRisksMsg = fmt.Sprintf("%s\n%s", acceptedRisksMsg, details)
```

Comment:
Coming into this line, `acceptedRisksMsg` is still the empty string from a few lines up.  Is there supposed to be a loop somewhere?  I don't understand the outgoing `Contains` check either.  But in [out-of-change CI][1] for this PR, you can see the opening newline due to this join:

```console
$ curl -s https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/origin-ci-test/pr-logs/pull/openshift_cluster-version-operator/841/pull-ci-openshift-cluster-version-operator-master-e2e-agnostic-upgrade-out-of-change/1575564385961644032/artifacts/e2e-agnostic-upgrade-out-of-change/gather-extra/artifacts/clusterversion.json | jq '.items[].status.history[].acceptedRisks' | uniq
"\nTarget release version..."
```

[1]: https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_cluster-version-operator/841/pull-ci-openshift-cluster-version-operator-master-e2e-agnostic-upgrade-out-of-change/1575564385961644032

Reply:
Yeah, that's a "legacy" issue that I blew off but I can go ahead fix it.